chris_backend/feeds/views.py

```diff
@@ -347,6 +347,8 @@ def perform_update(self, serializer):
         """
         Overriden to update feed's owners if requested by a PUT request.
         """
+        if not isinstance(self.request.data, dict) or not getattr(self.request.data, '_mutable', 1):
```

Comment:
Why did you merge ea06f75dcb3310cd7915d609a06a2fb2698235a0?

I think you should simply do a

```shell
git reset --hard 14f5281a46f79db29b0769b2279f36ea5e770f06
git push --force
```